### DISCUSSION TITLE
Apache 2.0 License choice?

### DISCUSSION THREAD

**Discussion by TimHeckel** [highlight/highlight#4376] [Q&A] [answered]
Thanks for sharing this! I'm curious why you chose the Apache 2.0 license over another possible license that would help mitigate cloud providers (read: AWS) simply taking this code and offering their own version of your service?

**Comment by jay-khatri** [comment, marked as answer]
Hey @TimHeckel, thanks for the message. We've definitely considered that. At our current stage, we're optimizing for making it as easy possible for any developer to adopt highlight at their respective company. And our decision on Apache 2 reflects that. 

We do have plans to have an `enterprise` directory long term where we license more sensitive parts of our code. But right now, we're more focused on adoption.

I hope that makes sense! Thanks again for reaching out.

**Reply by masterbater** [reply]
I hope you wont make a minio move.
